src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$color: #00acdc;
```

Comment:
The variable name '$color' is too generic, consider renaming it to something more descriptive, like '$primary-color'.